style.css

```diff
@@ -0,0 +1,44 @@
+html, body {
+  padding: 0;
+  margin: 0;  
+}
+
+nav {
+  display: inline;
+}
+
+div {
```

Comment:
Instead, this `div` should be `hamburger-wrapper`,`count-label-wrapper`,`cart-wrapper`. When you globally add style for `div` it will get you strange behavior in the future. Also, add `float: right` to `vertical-divider` if needed

Reply:
Also, you can save this `div`, if you add `nav` before him `nav div`